### PR TITLE
Add Home Assistant Icon Page to blueprint

### PR DESCRIPTION
**Pull request by bakito**
Add the Home Assistnt Icon page to the blueprint to be able to faster find the correct icons.

And a link to the color converter

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Ignore Mac DS_Store files
 .DS_Store
 **/.DS_Store
+.idea
```
Comment (albinmedoc) [inline .gitignore:4]: This is not needed

File: nspanel_blueprint.yaml
Comment (albinmedoc) [inline nspanel_blueprint.yaml:1]: Looks good

File: .gitignore (hunk 1 above)
Comment (bakito) [reply .gitignore:4]: It is needed to ignore IntelliJ idea project files.
If you could add this It would make my live easier.

Comment (albinmedoc) [reply .gitignore:4]: I don't have a problem adding it to the code. But it's not related to this PR. 

Maybe we could create a new PR and also ignore files for VS Code

Comment (bakito) [reply .gitignore]: I removed it

**Review by albinmedoc — APPROVED** [review]

**Pull request merged**